native/source/mixpanel/detail/mixpanel.cpp

```diff
@@ -53,11 +51,6 @@ namespace mixpanel
         ,min_log_level(LogEntry::LL_WARNING)
 #endif
     {
-        if (worker)
```

Comment:
No need for this since it is a member, and will always be `NULL` during initialization. Note that this *does* cause the multiple instances test to fail now.